src/sentry/static/sentry/app/stores/repositoryStore.tsx

```diff
@@ -0,0 +1,88 @@
+import Reflux from 'reflux';
+
+import RepoActions from 'app/actions/repositoryActions';
+import {Repository} from 'app/types';
+
+type RepositoryStoreInterface = {
+  get(
+    orgSlug?: string
+  ): {
+    repos: Array<Repository> | undefined;
+    reposLoading: boolean | undefined;
+    reposError: Error | undefined;
+  };
+
+  orgSlug: string | undefined;
+  lastUpdated: number;
+  repos: Array<Repository> | undefined;
+  reposLoading: boolean | undefined;
+  reposError: Error | undefined;
+
+  loadRepos(orgSlug: string): void;
+  loadReposSuccess(data: Array<Repository>): void;
+  loadReposError(error: Error): void;
+};
+
+const RepositoryStoreConfig: Reflux.StoreDefinition & RepositoryStoreInterface = {
+  orgSlug: undefined,
+  lastUpdated: Date.now(), // withRepositories will refetch if data is older than 30s
```

Comment:
If you're initializing with this value, doesn't that mean if your first fetch is within 30 seconds of loading, it won't load anything?

Reply:
I think billy's right, shouldn't this initialize to 0?

Reply:
Removed `lastUpdated`.